staging/src/k8s.io/component-base/featuregate/feature_gate.go

```diff
@@ -236,9 +236,9 @@ func (f *featureGate) SetFromMap(m map[string]bool) error {
 		}
 
 		if featureSpec.PreRelease == Deprecated {
-			klog.Warningf("Setting deprecated feature gate %s=%t. It will be removed in a future release.", k, v)
+			klog.InfoS("Setting deprecated feature gate. It will be removed in a future release.", "featureGate", k, "value", v)
 		} else if featureSpec.PreRelease == GA {
-			klog.Warningf("Setting GA feature gate %s=%t. It will be removed in a future release.", k, v)
+			klog.InfoS("Setting GA feature gate. It will be removed in a future release.", "featureGate", k, "value", v)
```

Comment:
Please take these changes out of your PR and focus on pkg/controller/podautoscaler.

Reply:
Done.